README.md

```diff
@@ -102,6 +103,25 @@ curl localhost:8080/version
 
 Stop ngsa by typing Ctrl-C or the stop button if run via F5
 
+### [Alternative to secrets] Visual Studio: CosmosDB access using Identity
+
+Below will illistrate how to add your user's principal Id to the correct group so that local development can take advantage of the managed identity.  
+
+In bash add your AAD user to CosmosDB:
+
+```bash
+# Get your own Principal ID (replace the email with yours)
+export PRINCIPAL=$(az ad user show --id YOUR-MSFT-ID@microsoft.com --query 'id' -o tsv)
+
+export COSMOS_RG=rg-ngsa-asb-dev-cosmos
+export COSMOS_NAME=ngsa-asb-dev-cosmos
+export COSMOS_SCOPE=$(az cosmosdb show -g $COSMOS_RG -n $COSMOS_NAME --query id -o tsv)
+
+# Add yourself to CosmosDB SQL Access
+az cosmosdb sql role assignment create -g $COSMOS_RG --account-name $COSMOS_NAME --role-definition-id 00000000-0000-0000-0000-000000000002 --principal-id $PRINCIPAL --scope $COSMOS_SCOPE
```

Comment:
Update instructions to create a user-assigned managed identity (UAMI) and a role assignment enabling the UAMI to access the cosmos DB

Reply:
Resolved. The instructions are fine as is for local runs, but in order to prevent too many role assignments from being created when multiple users run it locally, use a role assignment on a group, rather than multiple role assignments on single users. Instructions for adding the role assignment to the NGSA UAMI will be added to the ngsa-asb repository.

Reply:
I have converted a note on this to an issue so that we can cover it.